### PR TITLE
feat: Add Comments tab to User Profile

### PR DESCRIPTION
**Pull request by LulaV14**
image

### CODE REVIEW

File: packages/backend/src/services/user.service.ts [hunk 1]
```diff
@@ -284,4 +284,21 @@ export class UserService {
...
     return result.map((row: any) => ({ value: row.team as string, occurrences: Number.parseInt(row.count) }));
   }
+
+  async getUserComments(user: User, connectionArgs: ConnectionArgs): Promise<CommentConnection> {
+    logger.silly('[USER.SERVICE] userComments');
+
+    const userComments = await Comment.query()
+      .where('authorId', user.userId)
+      .innerJoin('insight', 'comment.insightId', 'insight.insightId')
+      .whereNull('insight.deletedAt')
```
Comment (baumandm) [inline packages/backend/src/services/user.service.ts:294]: Probably need to add `.whereNull('comment.deletedAt')` here.

File: packages/backend/schema.gql [hunk 2]
```diff
@@ -127,7 +127,7 @@ type CommentConnection {
 }
 
 type CommentEdge {
-  cursor: String!
+  cursor: String
```
Comment (baumandm) [inline packages/backend/schema.gql:130]: How did this change happen?

File: packages/frontend/src/components/insight-list/insight-list.tsx [hunk 3]
```diff
@@ -86,7 +86,7 @@ export const InsightList = ({
     );
   }
 
-  if (insightConnection.edges.length === 0) {
+  if (insightConnection.edges?.length === 0) {
```
Comment (baumandm) [inline packages/frontend/src/components/insight-list/insight-list.tsx:89]: `edges` are not nullable, so not sure why this is needed?

Comment (LulaV14) [reply packages/frontend/src/components/insight-list/insight-list.tsx:89]: you're right, I'm not sure how I suddenly got an error from here, seems like `insightConnection` shouldn't be undefined either at this point.

But I couldn't replicate it anymore, so maybe I'll just remove it from here and create a ticket if it happens again

File: packages/backend/schema.gql (hunk 2 above)
Comment (LulaV14) [reply packages/backend/schema.gql:130]: This was probably done when I was testing, but I'm not sure why it didn't revert back.

**Review by baumandm — APPROVED** [review]
Looks good but there's still one lint warning in `user.service.ts`

**Pull request merged**